nncf/common/tensor_statistics/statistic_point.py

```diff
@@ -71,6 +71,22 @@ def add_statistic_point(self, statistic_point: StatisticPoint) -> None:
 
             self.data[target_node_name].append(statistic_point)
 
+    def remove_statistic_point(self, algorithm: str) -> None:
+        """
+        Method to remove statistic point associated with a given algorithm
+        from statistic point container.
+
+        :param algorithm: Algorithm name
+        """
+        for target_node_name, statistic_points in list(self.data.items()):
+            # Reassign every target node name IF it doesn't contain the given algorithm
+            self.data[target_node_name] = [
```

Comment:
In this reassignment you forgot to directly delete the statistic points, which are not used anymore. I would expect to have it

Reply:
Good evening!
I am a bit confused, doesn't it get deleted with that re-assignment? 
Or do you want me to remove the associated algorithm from `statistic_point.algorithm_to_tensor_collectors` dict? 
Please elaborate more if possible